infra/prod/_modules/functions/function_op.tf

```diff
@@ -0,0 +1,86 @@
+locals {
+  openid_provider_func_settings = {
+    for s in var.op_func.app_settings :
+    s.name => s.key_vault_secret_name != null ? "@Microsoft.KeyVault(VaultName=${var.product}-kv;SecretName=${s.key_vault_secret_name})" : s.value
+  }
+}
```

Comment:
maybe we can include this "mapping" into the appropriate `_module` ?

Reply:
Ditto, this is similar to rp-func